hightime/_datetime.pyi

```diff
@@ -0,0 +1,93 @@
+import datetime as std_datetime
+from typing import Any, Optional, SupportsIndex, overload
+
+import hightime
+
+class datetime(std_datetime.datetime):
+    def __add__(self, other: std_datetime.timedelta, /) -> datetime: ...
+    def __eq__(self, other: object, /) -> bool: ...
+    def __ge__(self, other: std_datetime.date, /) -> bool: ...
+    def __gt__(self, other: std_datetime.date, /) -> bool: ...
+    def __hash__(self) -> int: ...
+    def __le__(self, other: std_datetime.date, /) -> bool: ...
+    def __lt__(self, other: std_datetime.date, /) -> bool: ...
+    def __ne__(self, other: object, /) -> bool: ...
+    @overload
+    @staticmethod
+    def __new__(
+        cls,
+        year: SupportsIndex,
```

Comment:
It turns out that this was the result of a compatibility-breaking change in Python 3.10: 
https://github.com/python/cpython/pull/15636

This became more obvious when I tried running the existing tests on Python 3.10-3.13, because `datetime.datetime` no longer accepts `IntLike`.

Reply:
https://github.com/ni/hightime/pull/44